src/frontend/screens/Game/GamePage/index.tsx

```diff
@@ -348,144 +357,86 @@ export default React.memo(function GamePage(): JSX.Element | null {
     const hasRequirements = extraInfo ? extraInfo.reqs.length > 0 : false
 
     return (
-      <div className="gameConfigContainer">
-        {!!(art_background ?? art_cover) &&
-          experimentalFeatures.enableNewDesign && (
-            <CachedImage
-              src={art_background || art_cover}
-              className="backgroundImage"
+      <SettingsContext.Provider value={settingsContextValues}>
+        <div className="gameConfigContainer">
```

Comment:
the diff here is a mess, but I'm just wrapping things in a SettingsContext wrapper, nothing changed inside this